docs/e2e.md

```diff
@@ -1,6 +1,10 @@
 # System / End 2 End test
 System Testing is a level of the software testing where a complete and integrated software is tested from an enduser perspective.
 
+## protractor in after-work.js 2.0.0
+The protractor module has been removed in after-work.js 2.0.0 and is needed to be included as a devDependencies. The reason is  mainly to keep the size of
+after-work.js down. If you use npm < 3.x the structure of node_modules will require you to also specify mocha as a devDependencies. 
```

Comment:
Mocha will be "installed" under node_modules/after-work.js/node_modules/mocha if you use npm 2.x and protractor in /node_modules/protractor and won't find mocha